content/user/apps/DiscoverVulns.md

```diff
@@ -13,14 +25,56 @@ With this list of discovered vulnerabiities, Contrast provides users extremely d
 * Line of code
 * Ability to replay the attack
 
-## How Vulnerabilities Work
-If a vulnerability is reported and Contrast has never seen it before, Contrast creates a new vulnerability. However, if that vulnerability already exists, Contrast updates the existing entry, issue count and number of days since it was last detected.
+## Export Findings
 
-<a href="assets/images/Application-vulnerabilities-tab.png" rel="lightbox" title="Vulnerability Details"><img class="thumbnail" src="assets/images/Application-vulnerabilities-tab.png"/></a>
+Export details on vulnerability findings from the **Vulnerabilities** page, a vulnerability's **Overview** page, or an application or server's **Vulnerabilities** tabs. Click the **Export** icon to choose either CSV or XML formats for the grouping of libraries that you want to include in the report.
```

Comment:
Same comment as earlier re: "Overview page". It's not actually labeled or titled that, it's just how we reference it, so to me this feels like a label I should be looking for.